umibukela/templates/partner_detail.html

```diff
@@ -55,16 +55,16 @@
     <div class="container">
       <div class="row">
         <div class="col-md-8">
-          <p><strong>The Folweni Community Resource Centre works to improve the quality of live of the Folweni community and the surrounding areas.</strong></p>
-          <p>To promote its initiatives, Folweni CRC works closely with government departments, community groups, non-governmental organizations, churches, small entrepreneurs and the corporate sector.</p>
+          <p><strong>{{ partner.intro_title }}</strong></p>
+          <p>{{ partner.intro_statement }}</p>
         </div>
         <div class="col-md-4 text-center">
-          <img src="{% static 'img/community-women.jpg' %}" class="img-responsive img-rounded-border">
+          <img src="{{ partner.intro_image.url }}" class="img-responsive img-rounded-border">
```

Comment:
I think there's a reasonable chance that for some partners we won't have images. So this field should be optional, and we should only show it if it's present. Ditto for the other image field.